module/VuFind/src/VuFind/Search/Base/Options.php

```diff
@@ -1286,11 +1288,11 @@ public function getAPISettings()
      * or side) and the value is the settings found in the file (which may be either
      * a single string or an array of strings).
      *
-     * @param string $handler Name of handler for which to load specific settings.
+     * @param ?string $handler Name of handler for which to load specific settings.
```

Comment:
We should specify default null behavior too:

```suggestion
     * @param ?string $handler Name of handler for which to load specific settings (null to load generic defaults).
```